### ISSUE TITLE
State restoring

### ISSUE THREAD

**Issue by WildOrangutan** [hbb20/CountryCodePickerProject#387]
I think it would be nice to implement state restoring, to restore selected country/code on rotation.

**Comment by hbb20** [comment]
Absolutely... Meanwhile, if you want apply ```app:ccp_rememberLastSelection="true"``` that should preserve selection on rotation using shared preferences.
